app/components/CaseStudyGallery.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import Image from "next/image";
+import Link from "next/link";
+
+import { featuredProjectsList } from "@/constants";
+import { laptopTemplate } from "@/public/png-icons/featured-projects-images";
+
+const CaseStudyGallery = () => {
+  return (
+    <section className="flex w-full max-w-[78rem] flex-col items-center justify-center gap-4 self-center px-6 py-12 md:gap-8 md:py-[4.5rem] xl:grid xl:grid-cols-2 xl:px-2">
+      {featuredProjectsList.map((project) => (
+        <Link
```

Comment:
This is an excellent example of a way we can refactor our components. Could you take a look at your Link component and see if you can refactor it? This makes the code more modular and easier to manage.

Maybe something similar to:  `<ProjectCard key={project.title} project={project} />`

Reply:
@cdsalty I have turned this into a reusable component. Working like a charm